src/styles/variables/_icons.scss

```diff
@@ -11,5 +11,11 @@ $icon-arrow-link: "\f0a9";
 $icon-asterisk: "\f069";
 $icon-check: "\F00C";
 $icon-home: "\f015";
-$icon-site-nav-default-size:30px;
-$icon-arrow-default-size: 24px;
+$icon-calendar: "\f073";
+$icon-clock: "\f017";
+$icon-download: "\f019";
+$icon-share: "\f14d";
+$icon-reminder: "\f849";
+$icon-register: "\f234";
+$icon-site-nav-default-size: 30px;
```

Comment:
Wondering if it makes sense to place these towards the top (line 6 and 7, where 8 is a space) since it would group them with the other icon sizing variables. They seem a bit out-of-place grouped with the unicode variables.

Reply:
yeah I kind of agree. It flows better that way to me